docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+# OpenVINO conditional compilation for optimal binary size{#opevino_conditional_compilation_deploying_guide}
+
+@sphinxdirective
+
+Conditional compilation can significantly reduce the binary size of OpenVINO™ package by excluding unnecessary components for inference of particular models, which makes the application benefit the optimal binary size.
+
+.. important::
+
+    The cost is OpenVINO runtime will only run inference for these particular models and the device for which was applied in conditional compilation.
+
+
+More details can be found `conditional_compilation_guide <https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md>`__ and `conditional_compilation_developer_guide <https://github.com/openvinotoolkit/openvino/blob/master/src/common/conditional_compilation/docs/develop_cc_for_new_component.md>`__
+
+There are mainly 2 stages to get optimal binary size of OpenVINO package size with conditional compilation:
```

Comment:
There are two steps to reduce binary size of OpenVINO runtime library with conditional compilation.

Reply:
done